packages/runtime/src/plugins/useUpstreamRetry.ts

```diff
@@ -87,11 +87,13 @@ export function useUpstreamRetry<TContext extends Record<string, any>>(
         if (maxRetries > 0) {
           setExecutor(function (executionRequest: ExecutionRequest) {
             let retries = maxRetries + 1;
+            let currentMinRetryDelay = minRetryDelay;
             let executionResult: MaybeAsyncIterable<ExecutionResult>;
             function retry(): MaybePromise<
               MaybeAsyncIterable<ExecutionResult>
             > {
               retries--;
+              currentMinRetryDelay += minRetryDelay;
```

Comment:
This gives us linear backoff. The standard is more leaning towards exponential backoff, with a rate of something between 1 and 2.
```suggestion
              currentMinRetryDelay = currentMinRetryDelay * 2;
```